app/src/main/java/org/mozilla/fenix/library/history/HistoryFragment.kt

```diff
@@ -155,11 +159,13 @@ class HistoryFragment : LibraryPageFragment<HistoryItem>(), UserInteractionHandl
             true
         }
         R.id.delete_history_multi_select -> {
+            val message = getMultiSelectSnackBarMessage(selectedItems)
             viewLifecycleOwner.lifecycleScope.launch(Main) {
                 deleteSelectedHistory(historyStore.state.mode.selectedItems, requireComponents)
                 viewModel.invalidate()
                 historyStore.dispatch(HistoryFragmentAction.ExitDeletionMode)
             }
+            showSnackBar(view!!, message)
```

Comment:
nit - can you move this inside the viewLifecycleOwner.lifecycleScope launch? It _should_ ensure there's a view that exists when we hit this

Reply:
Agreed, moved it inside the scope.